app/src/main/res/anim/slide_out_left.xml

```diff
@@ -0,0 +1,6 @@
+<set xmlns:android="http://schemas.android.com/apk/res/android">
+  <translate android:fromXDelta="0"
+    android:toXDelta="-100%"
+    android:duration="@integer/duration_anim_300_ms"/>
```

Comment:
Space issue: ` />`
Suggest using `Option+cmd+L` in all files to automatically resolve everything.

Reply:
Done